nemo/collections/asr/parts/submodules/ctc_beam_decoding.py

```diff
@@ -186,13 +188,14 @@ def __init__(
         self,
         blank_id: int,
         beam_size: int,
-        search_type: str = "default",
+        search_type: str = "beam",
         return_best_hypothesis: bool = True,
         preserve_alignments: bool = False,
         compute_timestamps: bool = False,
         beam_alpha: float = 1.0,
         beam_beta: float = 0.0,
-        kenlm_path: str = None,
+        word_kenlm_path: str = None,
```

Comment:
Can we leave `kenlm_path`, but add a separate parameter that indicates the type of kenlm model?
I think it will be more readable and extendable. 
For now, if I correctly understand, both parameters can be used, and only one silently will be used.

Reply:
Also, please, support `pathlib.Path` type for paths if possible